tool/minimap.re

```diff
@@ -52,7 +74,7 @@ static MINIMAP_COMPONENT = Component {
     },
     tick_mode: TickMode::DontCare,
     requested_delta_time: Option::None,
-    on_tick: fn() {},
+    on_tick: fn() { MINIMAP_STATE.calculate_minimap_size(MINIMAP_STATE.size); },
```

Comment:
I think it would be nicer if we only recalculated the minimap size when it was really necessary, which is either when the game resolution was changed (see the UE function in #178 which we can hook), or when its position was changed (in the menu code below).

Reply:
Currently I don't see any way of being able to hook the function without moving `MinimapState::calculate_minimap_size` fn into a Rust function.